inlong-dashboard/src/metas/sinks/defaults/MySQL.ts

```diff
@@ -120,26 +108,34 @@ export default class HiveSink extends SinkInfo implements DataWithBackend, Rende
   enableCreateResource: number;
 
   @FieldDecorator({
-    type: 'input',
-    rules: [{ required: true }],
-    props: values => ({
-      disabled: [110, 130].includes(values?.status),
-    }),
-  })
-  @ColumnDecorator()
-  @I18n('meta.Sinks.Username')
-  username: string;
-
-  @FieldDecorator({
-    type: 'password',
+    type: 'select',
     rules: [{ required: true }],
     props: values => ({
+      showSearch: true,
       disabled: [110, 130].includes(values?.status),
+      options: {
+        requestService: keyword => ({
```

Comment:
You may need to add `requestTrigger` here. Otherwise, the API will not be used when I enter keywords